### ISSUE TITLE
Add fingerprint modal on group join success

### ISSUE THREAD

**Issue by ekelen** [berty/berty#2219]
- [X] Display Fingerprint modal on group invitation QR scan to confirm group join
- [X] No modal if there is an error parsing invite or an error on group join

Make a new issue:
- [X] Include 'join'/'cancel' action buttons in this fingerprint popup (and move error on group join UX only after confirming join) - #2224

**Comment by moul** [comment]
🕚 Closing as no recent updates. Please reopen if you need any updates to this ticket.
